PROJECT

```diff
@@ -43,4 +43,20 @@ resources:
   kind: FlagSourceConfiguration
   path: github.com/open-feature/open-feature-operator/apis/core/v1alpha3
   version: v1alpha3
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: openfeature.dev
+  group: core
+  kind: FeatureFlag
```

Comment:
My only hesitation of this is the pluralization. It makes it sound like a single flag, which of course it isn't.

Is there a precedent for a `kind` with a plural name: ie `FeatureFlags`?

Reply:
There's a discussion going on here [1] regarding the naming and I think we can't do much here as `kind` requires the singular name (see names section explanation [2])

Another alternative is to use a suffix like `cfg` - `FeatureFlagCfg` but that's up to discussion. 

[1] - https://github.com/open-feature/open-feature-operator/issues/352
[2] - https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition

Reply:
I agree with this naming then, and I'm quite happy with it now.